vpr/SRC/route/route_tree_timing.c

```diff
@@ -418,7 +432,7 @@ update_unbuffered_ancestors_C_downstream(t_rt_node * start_of_new_path_rt_node)
 	return (rt_node);
 }
 
-static void load_rt_subtree_Tdel(t_rt_node * subtree_rt_root, float Tarrival) {
+void load_route_tree_Tdel(t_rt_node * subtree_rt_root, float Tarrival) {
```

Comment:
It's called in route_timing.c to calculate time delays of the pruned tree (initially a skeleton)